client/src/pages/tasks/Show.js

```diff
@@ -153,6 +182,23 @@ const BaseTaskShow = ({ pageDispatchers, currentUser }) => {
           position => currentUser.position.uuid === position.uuid
         )
       ))
+  const assessmentPeriods = [
+    {
+      start: moment().subtract(2, "months").startOf("month"),
+      end: moment().subtract(2, "months").endOf("month"),
```

Comment:
Race condition: start and end could be in different months. Define a `const now = moment()` and use that (here and below).

Reply:
done